src/app/container/version-modal/version-modal.component.html

```diff
@@ -13,433 +13,435 @@
-            title="Docker Image ID."
-            placeholder="e.g. 28d30887157514b6047c9"
-            [disabled]="true"
-          />
-        </div>
-      </div>
-      <div class="form-group form-group-sm">
-        <label class="col-sm-3 col-md-3 col-lg-3 control-label">
-          Dockerfile:
-        </label>
-        <div class="col-sm-9 col-md-9 col-lg-9">
-          <input
-            type="text"
-            class="form-control"
-            #name="ngModel"
-            id="dockerfilePath"
-            name="dockerfilePath"
-            [(ngModel)]="unsavedVersion.dockerfile_path"
-            minlength="3"
-            maxlength="256"
-            [pattern]="validationPatterns.dockerfilePath"
-            required
-            title="Relative path to the Dockerfile in the Git repository."
-            placeholder="e.g. /Dockerfile"
-            [disabled]="!editMode || tool?.mode === DockstoreToolType.ModeEnum.HOSTED"
-          />
-          <div *ngIf="formErrors.dockerfilePath" class="form-error-messages alert alert-danger">
-            {{ formErrors.dockerfilePath }}
-          </div>
-        </div>
-      </div>
-      <div class="form-group form-group-sm">
-        <label class="col-sm-3 col-md-3 col-lg-3 control-label">
-          CWL Descriptor:
-        </label>
-        <div class="col-sm-9 col-md-9 col-lg-9">
-          <input
-            type="text"
-            class="form-control"
-            name="cwlPath"
-            [(ngModel)]="unsavedVersion.cwl_path"
-            minlength="3"
-            maxlength="128"
-            [pattern]="validationPatterns.cwlPath"
-            [required]="!(unsavedVersion.cwl_path?.length || unsavedVersion.wdl_path?.length)"
-            title="Relative path to the CWL Descriptor in the Git repository."
-            placeholder="e.g. /Dockstore.cwl"
-            [disabled]="!editMode || tool?.mode === DockstoreToolType.ModeEnum.HOSTED"
-          />
-          <div *ngIf="formErrors.cwlPath" class="form-error-messages alert alert-danger">
-            {{ formErrors.cwlPath }}
+<div cdkDrag cdkDragRootElement=".cdk-overlay-pane">
+  <h4 mat-dialog-title>{{ TagEditorMode[mode] }} Version Tag</h4>
+  <div mat-dialog-content>
+    <app-alert></app-alert>
+    <form *ngIf="unsavedVersion" #tagEditorForm="ngForm" class="form-horizontal" name="tagEditorForm">
+      <div class="modal-body tagEditor">
+        <div class="form-group form-group-sm">
+          <label class="col-sm-3 col-md-3 col-lg-3 control-label">
+            Version Tag:
+          </label>
+          <div class="col-sm-9 col-md-9 col-lg-9">
+            <input
+              id="versionTagInput"
+              type="text"
+              class="form-control"
+              name="name"
+              [(ngModel)]="unsavedVersion.name"
+              ng-maxlength="128"
+              required
+              title="Docker Image tag name."
+              placeholder="e.g. develop"
```

Comment:
Nitpick, maybe the placeholder should be an actual version like `1.0` to encourage people to do the right thing